### ISSUE TITLE
cifar100 pretrained model

### ISSUE THREAD

**Issue by federicorossifr** [tensorflow/tpu#931]
I am trying to reproduce the results on cifar100 without success. 
Could it be possible to provide the pretrained model that has the accuracy described in the [paper ](https://arxiv.org/abs/1905.11946v5)(91.7%)?
